Swift/ReplicatorConfiguration.swift

```diff
@@ -354,4 +465,17 @@ public struct ReplicatorConfiguration {
         }
     }
     
+    mutating func initCollectionConfigs() {
+        if let db = _db {
+            guard let col = try? db.defaultCollection() else {
+                Database.throwNotOpenError()
+            }
+            
+            let colConfig = CollectionConfiguration()
+            addCollection(col, config: colConfig)
+        }
+    }
+    
+    var _collectionConfigs = [Collection: CollectionConfiguration]()
+    var _db: Database?
```

Comment:
I think we should stop using _ at some point as it's objective-c not swift.

Reply:
yea. we can update to avoid using underscore('_')